src/commands/mod.rs

```diff
@@ -267,7 +267,7 @@ impl EdgeAppManifest {
     pub fn save_to_file(manifest: &EdgeAppManifest, path: &Path) -> Result<(), CommandError> {
         let yaml = serde_yaml::to_string(&manifest)?;
         let manifest_file = File::create(path)?;
-        write!(&manifest_file, "{yaml}")?;
+        write!(&manifest_file, "---\n{}", yaml)?;
```

Comment:
they still should be ideally in this PR - so that we have a test for modified behavior. Those tests in that task will not be related. And that "linting" I don't think is in any way related to yaml linter. That linter will validate if the values used are correct.

Reply:
SO my point is that the second task is completely unrelated - save_to_file - there should not be modified at all as well

Reply:
> they still should be ideally in this PR - so that we have a test for modified behavior. Those tests in that task will not be related. And that "linting" I don't think is in any way related to yaml linter. That linter will validate if the values used are correct.

They will be

Reply:
Ok, so it's much simpler then.

I'll add tests and re-request the review.